contrib/ci/ctokenizer.py

```
@@ -87,7 +87,7 @@ def _token_fuzzy_match(token: Token, data: str) -> bool:
     """
 
     # hint
-    query: list[str] = data.split("@")
+    query: List[str] = data.split("@")
```

Ruff (0.15.14)

[error] Undefined name `List`

(F821)